ood-portal-generator/lib/ood_portal_generator/application.rb

```diff
@@ -73,6 +79,14 @@ def sum_path
         ENV['SUM'] || '/etc/ood/config/ood_portal.sha256sum'
       end
 
+      def dex_config
+        ENV['DEX_CONFIG'] || "/etc/ood/dex/config.yaml"
+      end
+
+      def dex_config_bak
+        "#{dex_config}.#{Time.now.strftime('%Y%m%dT%H%M%S')}"
```

Comment:
We should memoize this if the intention that during the single execution of this program a calls to dex_config_bak will produce the same result.

```
def dex_config_bak
  @dex_config_bak ||= ...
```

Reply:
This is done.